proto/panacea/datadeal/v2alpha2/datasale.proto

```diff
@@ -41,4 +41,7 @@ enum DataSaleStatus {
 
   // DATA_SALE_STATUS_DELIVERY_FAILED defines that the status of DataSale is delivery failed.
   DATA_SALE_STATUS_DELIVERY_FAILED = 5;
+
+  //DATA_SALE_STATUS_DEAL_FULL defines that the status of Deal associated with DataSale is full.
+  DATA_SALE_STATUS_DEAL_FULL = 6;
```

Comment:
IMO, It doesn't matter. But I think that you can change this name to `DATA_SALE_STATUS_DEAL_COMPLETED`.

Reply:
Okay, That seems to be a clearer expression. I change it.